docker/pytorch/cpu-blis/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# syntax = docker/dockerfile:experimental
+#
+# NOTE: To build this you will need a docker version > 18.06 with
+#       experimental enabled and DOCKER_BUILDKIT=1
+#
+#       For reference:
+#           https://docs.docker.com/develop/develop-images/build_enhancements/
+#
+# This Dockerfile will build Docker Image with PyTorch + DNNL + AMD BLIS and Torchvision installed for CPU only
+#
+# Example commandline to build PyTorch with AMD BLIS
+# sudo DOCKER_BUILDKIT=1 docker build . -t docker-image-repo-name
+ARG BASE_IMAGE=ubuntu:18.04
+ARG PYTHON_VERSION=3.8
+
+FROM ${BASE_IMAGE} as dev-base
+CMD echo "Welcome to the PyTorch Docker Container!" && \
+    echo "Version of PyTorch Installed: " && python -c 'import torch; print(torch.__version__)' && \
+    echo "Version of Torchvision Installed: " && python -c 'import torchvision; print(torchvision.__version__)' && \
+    echo "LDD output showing successful linking with BLIS: " && ldd /opt/conda/lib/python3.8/site-packages/torch/_C.cpython-38-x86_64-linux-gnu.so && \
+    /bin/bash
```

Comment:
Thanks for the feedback. The docker file expects the BLIS changes to be integrated with the master branch. To test the docker, I have cloned my repository (**RUN git clone https://github.com/shrutiramesh1988/pytorch.git** instead of **RUN git clone https://github.com/pytorch/pytorch.git** ) and ran the below commands :
sudo DOCKER_BUILDKIT=1 docker build . -t pytorch-blis-docker
sudo docker run --name py-blis-dock-container -it pytorch-blis-docker
Below is the output that gets displayed:
![image](https://user-images.githubusercontent.com/47769726/112959377-fcda8780-9160-11eb-98a8-300a8530ab9d.png)

Reply:
CMD specifies what command to run within the container. In this Dockerfile CMD is used to print the welcome text as shown in the snapshot above and it can be at the beginning.